File: MyApplication/app/src/main/java/com/evelyne/labs/myapplication/MainActivity.java
package com.evelyne.labs.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.annotation.SuppressLint;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import com.evelyne.labs.myapplication.admin.AdminLogIn;
import com.evelyne.labs.myapplication.customer.SignUp;
import com.evelyne.labs.myapplication.serviceprovider.SpSignUp;

public class MainActivity extends AppCompatActivity {

    private TextView administrator, serviceprovider, customer;
    private Button customerBtn,adminBtn,servicepBtn;
    @SuppressLint("MissingInflatedId")

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        customerBtn = findViewById(R.id.customer);
        adminBtn = findViewById(R.id.admin);
        servicepBtn = findViewById(R.id.serviceprovider);
        //open register
        customerBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, SignUp.class);
                startActivity(intent);
            }
        });
        adminBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, AdminLogIn.class);
                startActivity(intent);
            }
        });
        servicepBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, SpSignUp.class);
                startActivity(intent);
            }
        });
    }
}

File: DiceRoller/app/src/main/java/com/evelyne/dice/diceroller/MainActivity.kt
package com.evelyne.dice.diceroller

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        //declare and initialize variables

        val buttonToast: Button = findViewById(R.id.button_toast)
        buttonToast.setOnClickListener { showToast()  }

        val buttonRoll: Button = findViewById(R.id.button_roll)
        buttonRoll.setOnClickListener { rollDice()  }

        val buttonCount: Button = findViewById(R.id.button_count)
        buttonCount.setOnClickListener { rollCount() }
    }
    //create method for toss
    private fun showToast(){
        Toast.makeText(this, "Ooh My, you have tossed me", Toast.LENGTH_SHORT).show()
    }
    private fun rollDice(){
        val generatedRandom = (1..6).random()
        val textViewResult: TextView = findViewById(R.id.text_result)
        textViewResult.text = generatedRandom.toString()
    }
    private fun rollCount(){
        val textViewAdd : TextView = findViewById(R.id.text_result)
         val result = textViewAdd.text.toString().toInt()+1

        textViewAdd.text= result.toString()
    }
}